<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecentWidgetItem</class>
 <widget class="QWidget" name="RecentWidgetItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>56</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>262</width>
    <height>56</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>262</width>
    <height>56</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame#frame:hover
{
	border:1px solid #FF00B1EA;
}

QToolTip
{
	background-color: rgba(0, 0, 0, 128);
	border: 2px solid green;
    border-radius: 4px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true">background:transparent;
border:none;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lastTimeLabel">
           <property name="minimumSize">
            <size>
             <width>128</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>128</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:transparent;
border:none;</string>
           </property>
           <property name="text">
            <string>Last Play Time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="statusLabel">
           <property name="styleSheet">
            <string notr="true">background:transparent;
border:none;</string>
           </property>
           <property name="text">
            <string>Status</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearButton">
           <property name="minimumSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="fileNameLabel">
        <property name="minimumSize">
         <size>
          <width>262</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>262</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;
border:none;</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::WidgetWidth</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
        <property name="text" stdset="0">
         <string>Video File Name</string>
        </property>
        <property name="wordWrap" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resource/player.qrc"/>
 </resources>
 <connections/>
</ui>
